docs/styles.css

```diff
@@ -2707,26 +2707,30 @@
   align-items: center;
   height: calc(100% + 1px);
   margin-bottom: -1px;
-  color: var(--psColorsGray03);
+  color: var(--psColorsGray02);
   font-weight: var(--psTypeFontWeightBook);
   background: none;
   border: 0;
   cursor: pointer;
   overflow: hidden;
-  padding-bottom: 4px;
+  padding: 0 0 var(--psLayoutSpacingTiny) 0;
+  transition: color var(--psMotionSpeedNormal);
 }
 .ps-tab-list-item___2FBfq + .ps-tab-list-item___2FBfq {
   margin-left: var(--psLayoutSpacingLarge);
 }
+.ps-tab-list-item___2FBfq:focus {
+  color: var(--psColorsWhite);
+  outline: 0;
```

Comment:
oh, I'm pointing at the `docs/` directory.  I shouldn't comment here, since this is just generated source.  But please apply these comments toward the `packages/` src files.